assets/js/module/iframePreview.js

```diff
@@ -63,4 +64,16 @@ export default class IframePreview {
             self.loadIframes(iframes)
         });
     }
+
+    #changeSelfTargetLinksToParent(body) {
+        const parser = new DOMParser()
+        const dom = parser.parseFromString(body, 'text/html')
+        ;[...dom.getElementsByTagName('a')].forEach((link) => {
+            if (!link.getAttribute('target')) {
+                link.setAttribute('target', '_parent')
+            }
+        })
+
+        return dom.documentElement.outerHTML
```

Comment:
I feel like I'll lean a lot why you. :-). Why no semi-colon at the end of lines?

What if a internal link has a target? Nevertheless it's good for a first implementation. Will see how it works

Reply:
- It's an habit but it's a good point, we need to specify our JS dev guidelines

- I was thinking that if target is declared  it will be with _blank value and make sense for me to keep the behavior. But we can talk about and I will make also a more precise condition

Reply:
Good idea, let's try eslint? https://eslint.org/

Reply:
👍